nevergrad/benchmark/experiments.py

```diff
@@ -228,10 +228,9 @@ def instrum_discrete(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                 else:
                     assert instrum_str == "Unordered"
                     instrum = ng.p.TransitionChoice(range(arity), repetitions=nv)
-                for discrete_func in [corefuncs.onemax, corefuncs.leadingones, corefuncs.jump]:
-                    dfunc = ExperimentFunction(discrete_func, instrum.set_name("noname"))
+                for name in ["onemax", "leadingones", "jump"]:
+                    dfunc = ExperimentFunction(corefuncs.DiscreteFunction(name, arity), instrum.set_name(instrum_str))
                     dfunc.add_descriptors(arity=arity)
-                    dfunc.add_descriptors(instrum_str=instrum_str)
```

Comment:
Parameters have a `set_name` method for this

Reply:
Thanks, I see that you fixed it by yourself.